google/cloud/spanner_dbapi/parse_utils.py

```diff
@@ -169,10 +160,7 @@ def parse_insert(insert_sql, params):
     if values.homogenous():
         # Case c)
 
-        columns = [
-            strip_backticks(mi.strip())
-            for mi in match.group("columns").split(",")
-        ]
```

Comment:
This `columns` list is not used in calculations, only its length. The column names themselves are only used in the "unequal length" error message